### PR TITLE
net-dialup/freeradius: Drop virtual/mysql from DEPEND

### PR DESCRIPTION
**Pull request by geaaru**
Bug: https://bugs.gentoo.org/665962
Closes: https://bugs.gentoo.org/665962
Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: net-dialup/freeradius/freeradius-3.0.17.ebuild [hunk 1]
```diff
@@ -214,7 +219,14 @@ src_install() {
 	systemd_newtmpfilesd "${FILESDIR}"/freeradius.tmpfiles freeradius.conf
 	systemd_dounit "${FILESDIR}"/freeradius.service
 
-	find "${ED}" \( -name "*.a" -o -name "*.la" \) -delete || die
```
Comment (Amynka) [inline net-dialup/freeradius/freeradius-3.0.17.ebuild]: We prefer this instead of prune_libtool_files

File: net-dialup/freeradius/freeradius-3.0.17.ebuild [hunk 2]
```diff
@@ -214,7 +219,14 @@ src_install() {
...
+	# Remove all default sites under site-enables.
+	# Administrator install all sites when configure freeradius.
+	rm "${D}"/etc/raddb/sites-enabled/{inner-tunnel,default}
```
Comment (Amynka) [inline net-dialup/freeradius/freeradius-3.0.17.ebuild]: missing die

File: net-dialup/freeradius/freeradius-3.0.17.ebuild [hunk 3]
```diff
@@ -214,7 +219,14 @@ src_install() {
...
+	# Remove part of modules that require a configuration
+	# from mods-enabled directory.
+	rm "${D}"/etc/raddb/mods-enabled/{cache_eap,digest,eap,mschap,ntlm_auth}
```
Comment (Amynka) [inline net-dialup/freeradius/freeradius-3.0.17.ebuild]: missing die

File: net-dialup/freeradius/freeradius-3.0.17.ebuild (hunk 1 above)
Comment (geaaru) [reply net-dialup/freeradius/freeradius-3.0.17.ebuild]: Ok, i revert it but personally i prefer use of a specific function from an eclass, more easy to maintains, imho.

File: net-dialup/freeradius/freeradius-3.0.17.ebuild (hunk 3 above)
Comment (geaaru) [reply net-dialup/freeradius/freeradius-3.0.17.ebuild]: why doesn't to permit an installation that works ? But then I didn't think that if a user upgrade package and he has a site active we remove it. My fault. I remove it. Thanks

File: net-dialup/freeradius/freeradius-3.0.17.ebuild [hunk 4]
```diff
@@ -21,10 +21,13 @@ SLOT="0"
 
 IUSE="
 	debug firebird iodbc kerberos ldap libressl memcached mysql odbc oracle pam
-	pcap postgres python readline rest samba sqlite ssl
+	pcap postgres python readline rest samba sqlite ssl redis
```
Comment (mgorny) [inline net-dialup/freeradius/freeradius-3.0.17.ebuild]: I don't really think adding USE=redis is related at all to virtual/mysql. It belongs in a separate commit. So does changing maintainer.

Comment (geaaru) [reply net-dialup/freeradius/freeradius-3.0.17.ebuild]: I seperate commit and fix redis library dep. thanks.

**Comment by geaaru** [comment]
@mgorny: I take a look to your note and currently REQUIRED_USE with ${PYTHON_REQUIRED_USE} is set when `python` use flag is set. Do you receive this message when `python` use flag is not set ? If yes in this way what is the best practice for handle this use case ?

Thanks in advance for any suggestion

**Comment by mgorny** [comment]
Ah. You need to call python..._pkg_setup conditionally to 'use python'
